qoibench.c

```diff
@@ -131,7 +132,7 @@ void libpng_encode_callback(png_structp png_ptr, png_bytep data, png_size_t leng
 	write_data->size += length;
 }
 
-void *libpng_encode(void *pixels, int w, int h, int channels, int *out_len) {
+static void *libpng_encode(void *pixels, int w, int h, int channels, int *out_len) {
```

Comment:
Prefer `size_t` for all usages as size/quantities, like `w`, `h`, `channels` and for `out_len`.